tests/pytests/pkg/downgrade/test_salt_downgrade.py

```diff
@@ -36,17 +36,25 @@ def test_salt_downgrade(salt_call_cli, install_salt):
     assert "Authentication information could" in use_lib.stderr
 
     # Verify there is a running minion by getting its PID
+    salt_name = "salt"
     if platform.is_windows():
         process_name = "salt-minion.exe"
     else:
         process_name = "salt-minion"
-    old_pid = None
+
+    old_pid = []
```

Comment:
Are we expecting more than one pid to act upon?

Reply:
allowing for possibility
```
[root@Unknown david]# systemctl restart salt-minion
[root@Unknown david]# ps -ef | grep salt-minion
root        4865       1  3 09:52 ?        00:00:00 /opt/saltstack/salt/bin/python3.10 /usr/bin/salt-minion
root        4873    4865 10 09:52 ?        00:00:00 /opt/saltstack/salt/bin/python3.10 /usr/bin/salt-minion MultiMinionProcessManager MinionProcessManager
root        4962    3611  0 09:52 pts/0    00:00:00 grep --color=auto salt-minion
[root@Unknown david]#
```